configure.ac

```diff
@@ -70,9 +70,6 @@ AC_CHECK_HEADER([google/protobuf/message.h],,
                 [AC_MSG_ERROR([protobuf headers could not be found])])
 AC_CHECK_HEADER([leveldb/db.h],,
                 [AC_MSG_ERROR([leveldb headers could not be found])])
```

Comment:
Don't think leveldb is used now, so can go from here and gclient config?

Reply:
Gone!